### PR TITLE
feat(natds-js): add icons name and id swift output

### PR DESCRIPTION
**Pull request by arielwb**
affects: @naturacosmeticos/natds-icons

# Description

This pr adds a .swift file creation to the icons lib to help with ios usage.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Running the script
- [ ] Test B

**Test Configuration**:
* Node version: X.Y.Z
* Yarn version: X.Y.Z

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/icons/scripts/svgToFont.js [hunk 1]
```diff
@@ -8,6 +8,29 @@ const files = './src/assets/cleaned/**/*.svg';
...
 
+function createSwiftOutput(metadata) {
+  const capitalizeWord = word => `${word[0].toUpperCase()}${word.slice(1)}`;
+
+  const toCamelCase = word => word
+    .split('-')
+    .map((item, index) => index === 0 ? item : capitalizeWord(item))
+    .join('')
+
+  const toIosUnicode = (unicode) => `${unicode.replace('%u', '\\u{')}}`
```
Comment (viniciusponciano-tw) [inline packages/icons/scripts/svgToFont.js:19]: Me parece que ficou faltando fechar a chave aberta neste trecho do replace `'\\u**{**'`.

Faz sentido?

Comment (jimmyandrade) [reply packages/icons/scripts/svgToFont.js:19]: Também tive essa mesma impressão.

File: packages/icons/scripts/svgToFont.js [hunk 2]
```diff
@@ -8,6 +8,29 @@ const files = './src/assets/cleaned/**/*.svg';
 const fontName = 'natds-icons';
 const types = ['eot', 'ttf', 'woff', 'woff2', 'svg'];
 
+function createSwiftOutput(metadata) {
```
Comment (jimmyandrade) [inline packages/icons/scripts/svgToFont.js:11]: Desculpem a falta de contexto: por que estamos gerando _output_ para Swift a partir de um pacote do web?

File: packages/icons/src/natds-icons.swift [hunk 3]
```diff
@@ -0,0 +1,179 @@
+
```
Comment (jimmyandrade) [inline packages/icons/src/natds-icons.swift:1]: Esse arquivo é gerado pelo `svgToFont.js`? Se sim, ele deveria estar versionado mesmo?

File: packages/icons/scripts/svgToFont.js (hunk 2 above)
Comment (jimmyandrade) [reply packages/icons/scripts/svgToFont.js:11]: Outra questão: essa função tem um teste relacionado? Se mexermos no corpo dessa função, algum teste quebra?

**Pull request merged**